packages/shared/src/components/modals/award/GiveAwardModal.tsx

```diff
@@ -555,6 +557,7 @@ type GiveAwardModalProps = ModalProps & {
   type: AwardTypes;
   entity: AwardEntity;
   post?: Post;
+  flags?: Record<string, string>;
```

Comment:
Is this going to be updated into an actual type to be certain of the properties?